service/service.go

```diff
@@ -139,6 +141,17 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.RuntimeMetricsCollector() == nil {
+		var runtimeMetricsConfig metrics.RuntimeConfig
+		err := svc.configProvider.Get("runtimeMetrics").PopulateStruct(&runtimeMetricsConfig)
```

Comment:
Who manages top level metrics? I don't want to accidentally override a preconfigured key for eg.